src/v/redpanda/application.h

```diff
@@ -80,34 +80,44 @@ class application {
     ss::future<> set_proxy_config(ss::sstring name, std::any val);
     ss::future<> set_proxy_client_config(ss::sstring name, std::any val);
 
+    smp_groups smp_service_groups;
+
+    // Sorted list of services (public members)
+    ss::sharded<archival::scheduler_service> archival_scheduler;
+
+    ss::sharded<cloud_storage::cache> shadow_index_cache;
+    ss::sharded<cloud_storage::partition_recovery_manager>
+      partition_recovery_manager;
+    ss::sharded<cloud_storage::remote> cloud_storage_api;
+
+    ss::sharded<cluster::id_allocator_frontend> id_allocator_frontend;
     ss::sharded<cluster::metadata_cache> metadata_cache;
-    ss::sharded<kafka::group_router> group_router;
-    ss::sharded<cluster::shard_table> shard_table;
-    ss::sharded<storage::api> storage;
-    ss::sharded<storage::node_api> storage_node;
-    std::unique_ptr<coproc::api> coprocessing;
-    ss::sharded<coproc::partition_manager> cp_partition_manager;
-    ss::sharded<cluster::partition_manager> partition_manager;
-    ss::sharded<raft::recovery_throttle> recovery_throttle;
-    ss::sharded<raft::group_manager> raft_group_manager;
     ss::sharded<cluster::metadata_dissemination_service>
       md_dissemination_service;
-    ss::sharded<kafka::coordinator_ntp_mapper> coordinator_ntp_mapper;
+    ss::sharded<cluster::partition_manager> partition_manager;
+    ss::sharded<cluster::rm_partition_frontend> rm_partition_frontend;
+    ss::sharded<cluster::shard_table> shard_table;
+    ss::sharded<cluster::tx_gateway_frontend> tx_gateway_frontend;
+
+    ss::sharded<coproc::partition_manager> cp_partition_manager;
+
     ss::sharded<kafka::coordinator_ntp_mapper> co_coordinator_ntp_mapper;
-    std::unique_ptr<cluster::controller> controller;
+    ss::sharded<kafka::coordinator_ntp_mapper> coordinator_ntp_mapper;
     ss::sharded<kafka::fetch_session_cache> fetch_session_cache;
-    smp_groups smp_service_groups;
+    ss::sharded<kafka::group_router> group_router;
     ss::sharded<kafka::quota_manager> quota_mgr;
-    ss::sharded<cluster::id_allocator_frontend> id_allocator_frontend;
-    ss::sharded<cloud_storage::remote> cloud_storage_api;
-    ss::sharded<cloud_storage::partition_recovery_manager>
-      partition_recovery_manager;
-    ss::sharded<archival::scheduler_service> archival_scheduler;
     ss::sharded<kafka::rm_group_frontend> rm_group_frontend;
-    ss::sharded<cluster::rm_partition_frontend> rm_partition_frontend;
-    ss::sharded<cluster::tx_gateway_frontend> tx_gateway_frontend;
+
+    ss::sharded<raft::group_manager> raft_group_manager;
+    ss::sharded<raft::recovery_throttle> recovery_throttle;
+
+    ss::sharded<storage::api> storage;
+    ss::sharded<storage::node_api> storage_node;
+
     ss::sharded<v8_engine::data_policy_table> data_policies;
-    ss::sharded<cloud_storage::cache> shadow_index_cache;
+
+    std::unique_ptr<cluster::controller> controller;
+    std::unique_ptr<coproc::api> coprocessing;
```

Comment:
Sorted the list of services (as we had agreed to for "lists of stuff", to reduce conflicts) and grouped by namespace. This is a separate change from the rest.

The order here doens't matter for `sharded<foo>` since the services actually get constructed later in a specific order in `wire_up_services`.